manage-engine-key-manager-plus/info.json

```diff
@@ -0,0 +1,222 @@
+      },
+      {
+        "title": "API Key",
+        "required": true,
+        "editable": true,
+        "visible": true,
+        "type": "password",
+        "name": "token",
+        "description": "API key that is configured for your account to access the Manage Engine Key Manager Plus endpoint.",
+        "tooltip": "API key that is configured for your account to access the Manage Engine Key Manager Plus endpoint."
+      },
+      {
+        "title": "Verify SSL",
+        "required": false,
+        "editable": true,
+        "visible": true,
+        "type": "checkbox",
+        "name": "verify_ssl",
+        "value": true,
+        "description": "Specifies whether the SSL certificate for the server is to be verified or not. By default, this option is set as True.",
+        "tooltip": "Specifies whether the SSL certificate for the server is to be verified or not. By default, this option is set as True."
+      }
+    ]
+  },
+  "operations": [
+    {
+      "operation": "get_ssh_keys",
+      "title": "Get SSH Keys",
+      "description": "Retrieves all SSH Keys across your network and import them into Key Manager Plus repository.",
+      "category": "investigation",
+      "annotation": "get_ssh_keys",
+      "output_schema": {
+          "name": "",
+          "result": {
+            "status": "",
+            "message": ""
+          },
+          "totalRows": "",
+          "details": [{
+            "KeyName": "",
+            "KeyType": "",
+            "KeyLength": "",
+            "FingerPrint": "",
+            "isPassphraseAvailable": "",
+            "CreatedBy": "",
+            "CreationTime": ""
+          }]
+      },
+      "enabled": true,
+      "parameters": []
+    },
+    {
+      "operation": "get_ssl_certificates",
+      "title": "Get SSL Certificates",
+      "description": "Retrieves all SSL certificates across your network and import them into Key Manager Plus repository.",
+      "category": "investigation",
+      "annotation": "get_ssl_certificates",
+      "output_schema": {
+         "name": "",
+         "totalRows": "",
+         "details": {}
+      },
+      "enabled": true,
+      "parameters": [
+        {
+          "title": "Get SSL",
+          "required": true,
+          "editable": true,
+          "visible": true,
+          "type": "select",
+          "options": [
+            "Hostname/IP Address",
+            "IP Address Range"
+          ],
+          "name": "search_type",
+          "value": "Hostname/IP Address",
+          "tooltip": "Select one of the method to get SSL certificates across your network and import them into Key Manager Plus repository.",
+          "description": "Select one of the method to get SSL certificates across your network and import them into Key Manager Plus repository. You can choose between Hostname/IP Address or IP Address Range.",
```

Comment:
As we are providing a default value of Hostname/IP Address can we same mention this in description.

Reply:
done